### DISCUSSION TITLE
Subscribe to webhook on deploy

### DISCUSSION THREAD

**Discussion by craveytrain** [beginner-corp/begin-community#105] [Ideas]
I am setting up a POST handler for a webhook and I would like to be able to subscribe to it automatically when the app is deployed. Is there a way to fire off some script/code on deployment (and ideally, on teardown as well)?

**Comment by tbeseda** [comment]
Hi @craveytrain -- thanks for stopping by
Unfortunately Begin doesn't have support for post deploy hooks yet. So it's not possible to run something when the deploy is finished.
I think it's a cool idea though! Plenty of use-cases I can think of. So I added the enhancement tag and moved this discussion to Ideas.
Let us know if we can help with anything else Begin or Architect.

**Reply by brianleroux** [reply]
yeah I love this idea; some time ago we did experiment with custom events prefixed with begin that the platform invokes. eg:
- `begin-deploy-before`
- `begin-deploy-after`

would this work ?
